nix/targets.nix

```diff
@@ -12,6 +12,7 @@ let
   status-go = callPackage ./status-go {
     inherit (mobile) xcodeWrapper;
     androidPkgs = mobile.android.androidComposition;
+    envFlags = callPackage ./tools/envParser.nix { };
```

Comment:
If this is used only by `status-go` why is it loaded here and passed via `callPackage`? Why not load it directly in `nix/status-go/default.nix`?

Reply:
I'm still not comfortable with the idea of referencing paths with `../` in Nix, it feels like I'm referencing paths that the current file isn't supposed to know about. But it does look better. Changed.